CHANGELOG.md

```diff
@@ -1,18 +1,30 @@
-[//]: # "START/LATEST"
-
+[//]: # (START/LATEST)
 # Latest
 
 ## Features
-
-- A user-friendly description of a new feature. {issue-number}
+  * A user-friendly description of a new feature. {issue-number}
 
 ## Fixes
-
-- A user-friendly description of a fix. {issue-number}
+ * A user-friendly description of a fix. {issue-number}
 
 ## Security
+ * A user-friendly description of a security fix. {issue-number}
+
+---
+
+[//]: # "START/v1.3.0"
+
+# v1.3.0
+
+## Features
+
+- The TOTP code of a TOTP field can now be accessed using the `.totp` property of a field. {#33}
+- Release of the SDK also includes `wheel` format. {#45}
```

Comment:
This is rather a fix for the following reasons:
- Before automated release (with a pipeline), we'd include the `wheel format` as part of the manual release process.
- When we switched to automated release, by accident we missed including it.

Reply:
Good catch! Removed it altogether since this means it effectively is an internal change.